SU2_CFD/include/drivers/CDriver.hpp

```diff
@@ -466,19 +466,29 @@ class CDriver : public CDriverBase {
   /*!
    * \brief Process the boundary conditions and update the multi-grid structure.
    */
-  void BoundaryConditionsUpdate();
+  void UpdateBoundaryConditions();
 
   /*!
-   * \brief Get the number of time iterations.
-   * \return Number of time iterations.
+   * \brief Update the geometry (i.e. dual grid).
    */
-  unsigned long GetNumberTimeIter() const;
+  void UpdateGeometry();
 
   /*!
-   * \brief Get the current time iteration.
-   * \return Current time iteration.
+   * \brief Update the primal far-field variables.
    */
-  unsigned long GetTimeIter() const;
+  void UpdateFarfield();
+
+  /*!
+   * \brief Get the number of external iterations.
+   * \return Number of external iterations.
+   */
+  unsigned long GetNumberTimeIterations() const;
+
+  /*!
+   * \brief Get the current external iteration.
+   * \return Current external iteration.
+   */
+  unsigned long GetTimeIteration() const;
```

Comment:
Can these and other similar ones be reverted/removed (viscosity, conductivity, etc.)?

Reply:
I did some cleanup. Any other ones I should remove? 

And `GetNumberTimeIterations()` and `GetTimeIteration()` are not new additions. I just renamed them into the long form to be consistent with other functions